src/lib/eventStack/eventStack.js

```diff
@@ -49,10 +46,12 @@ class EventStack {
     if (!isBrowser) return
 
     const { target = document, pool = 'default' } = options
-    const eventTarget = this._find(target)
+    const eventTarget = this._find(target, false)
```

Comment:
Before this change, `_find` created always new instance of `EventTarget` when it was removed